myapp/app/controllers/application_controller.rb

```diff
@@ -17,4 +17,11 @@ def require_login
   def current_user
     @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
   end
+
+  def require_admin
```

Comment:
Where is require_admin being called?

Reply:
I'll use it in a subsequent STEP, but it's not needed here. So I'll delete it and add it when I need it.